src/index.spec.js

```diff
@@ -580,15 +580,17 @@ describe('Dropzone', () => {
         dataTransfer: { files: images.concat(files) }
       })
       const rejected = dropSpy.firstCall.args[1]
-      expect(rejected.length).toEqual(2)
+      expect(rejected.length).toBeGreaterThan(1)
     })
 
     it('should allow single files to be dropped if multiple is false', () => {
-      const dropzone = mount(<Dropzone accept="image/*" onDrop={dropSpy} multiple={false} />)
+      const dropzone = mount(
+        <Dropzone accept="image/*" onDrop={dropAcceptedSpy} multiple={false} />
```

Comment:
Not sure why did you rename the spy name. This is a bit misleading now IMO since it includes both accepted and rejected files.

Reply:
gonna change it back